src/zenml/model/utils.py

```diff
@@ -183,3 +179,46 @@ def log_model_metadata(
         mv = ModelVersion(name=model_name, version=model_version)
 
     mv.log_metadata(metadata)
+
+
+def link_artifact_to_model(
+    artifact_version_id: UUID,
+    model_version: Optional["ModelVersion"] = None,
+    is_model_artifact: bool = False,
+    is_deployment_artifact: bool = False,
+) -> None:
+    """Link the artifact to the model.
+
+    Args:
+        artifact_version_id: The ID of the artifact version.
+        model_version: The model version to link to.
+        is_model_artifact: Whether the artifact is a model artifact.
+        is_deployment_artifact: Whether the artifact is a deployment artifact.
+
+    Raises:
+        RuntimeError: If called outside of a step.
+    """
+    if not model_version:
+        is_issue = False
+        try:
+            step_context = get_step_context()
+            model_version = step_context.model_version
+        except StepContextError:
+            is_issue = True
```

Comment:
what does `is_issue` mean or denote? issue as in problem? Was a bit unclear for me.

Reply:
Yep, issue like a problem